### PR TITLE
Prevent crash when using types with spreads

### PR DESCRIPTION
**Pull request by StevePotter**
Fixes https://github.com/gajus/eslint-plugin-flowtype/issues/293

This just avoids a crash.  It doesn't go any further.  But at least the plugin is usable :)

Oh man, I just saw https://github.com/gajus/eslint-plugin-flowtype/pull/266.  This might be a duplicate although that PR seems more ambitious so I'll leave this PR open

### CODE REVIEW

File: src/rules/noDupeKeys.js [hunk 1]
```diff
@@ -62,6 +62,10 @@ const create = (context) => {
 
     _.forEach(node.properties, (identifierNode) => {
       const needle = {name: getParameterName(identifierNode, context)};
+      // avoids error for types constructed from spreading others like: type Props = { ...A, ...B }
+      if (identifierNode.type === 'ObjectTypeSpreadProperty') {
+        return
```
Comment (serchavalos) [inline src/rules/noDupeKeys.js]: missing `;` bro 😄

**Comment by suchipi** [comment]
@gajus I know babel 7 is still beta, but is there any chance this could be merged before its release? This PR is not very intrusive.

**Comment by MoOx** [comment]
This is not tied to babel 7. You can use latest babel-eslint (8.x) with babel 6 and get this problem. This PR fix the bug I have. Would be nice to see this merged!

@StevePotter could you make CI green? There is some easy fix here https://travis-ci.org/gajus/eslint-plugin-flowtype/jobs/330540986#L2043

**Comment by kusmierz** [comment]
The same issue seems to appear for `sortKeys` rule:
```
Cannot read property 'type' of undefined
TypeError: Cannot read property 'type' of undefined
    at ./node_modules/eslint-plugin-flowtype/dist/rules/sortKeys.js:85:24
```

**Comment by IanVS** [comment]
https://github.com/gajus/eslint-plugin-flowtype/issues/293 is closed/fixed, so I think this PR can be closed as well now?  Although I don't know if `sortKeys` is still broken (I don't use it).

**Comment by gajus** [comment]
> Although I don't know if sortKeys is still broken (I don't use it).

That would be a separate issue anyway.

**Pull request closed without merging**